src/components/Activities/index.tsx

```diff
@@ -60,13 +60,13 @@ const Activities = () => {
             <Typography width="80%" textAlign="center">
               With{' '}
               <Typography fontWeight="bold" display="inline">
-                {'Safe {Pass}'}
+                {'Safe{Pass}'}
```

Comment:
If we're adding a space here, we should probably do it everywhere to be consistent. There's no space in the activities page, the title of the token locking page and the manifest.json